### PR TITLE
Fix multi-line strings in DeclareLaunchArgument

### PR DESCRIPTION
**Pull request by MatthijsBurgh**
Related to:
https://github.com/UniversalRobots/Universal_Robots_ROS2_Description/pull/140
https://github.com/UniversalRobots/Universal_Robots_ROS2_Driver/pull/948
https://github.com/UniversalRobots/Universal_Robots_ROS2_Ignition_Simulation/pull/16

### CODE REVIEW

**Comment by MatthijsBurgh** [comment]
Rolling industrial CI is broken. As it migrated to Noble. But no package have been released yet for Noble.

File: ur_simulation_gazebo/launch/ur_sim_control.launch.py
Comment (fmauch) [inline ur_simulation_gazebo/launch/ur_sim_control.launch.py:211]:
```suggestion
            description="Package with the controller\'s configuration in \"config\" folder. "
            "Usually the argument is not set, it enables use of a custom setup.",
```

File: ur_simulation_gazebo/launch/ur_sim_moveit.launch.py
Comment (fmauch) [inline ur_simulation_gazebo/launch/ur_sim_moveit.launch.py:117]:
```suggestion
            description="Package with the controller\'s configuration in \"config\" folder. "
            "Usually the argument is not set, it enables use of a custom setup.",
```

**Review by fmauch — COMMENTED** [review]
Thank you for your contribution. This indeed improves readability and has been forgotten since it had been initially created.

I think that's a good place to fix the mixed quotation style, as well, I've added suggestions for this.

**Comment by MatthijsBurgh** [comment]
@fmauch Need approval for the CI again.

**Review by fmauch — APPROVED** [review]
This looks good, thanks again @MatthijsBurgh . Will merge this once the non-ici jobs succeed.

Edit: Seems like the black formatter disagrees with my attempt to unify quotation styles.

**Pull request merged**
